.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "developers.home-assistant",
+    "dockerFile": "Dockerfile",
+    "appPort": [
+        3000
+    ],
+    "postCreateCommand": "yarn install",
+    "extensions": [
+        "davidanson.vscode-markdownlint",
+        "eamodio.gitlens",
```

Comment:
Heavy extension that isn't required to contribute.

Reply:
Agreed (but damn it's so nice!)